customization-base/src/main/java/com/azure/autorest/customization/ClassCustomization.java

```diff
@@ -571,18 +366,21 @@ public ClassCustomization removeAnnotation(String annotation) {
      * @return the current class customization for chaining
      */
     public ClassCustomization renameEnumMember(String enumMemberName, String newName) {
```

Comment:
I assume it could have problem, if the Enum member is used somewhere.

Though, so far, I cannot think of a case that the Enum member is reference in generated code (except sample/test).